addons/medical_treatment/data/model.cfg

```diff
@@ -58,6 +58,8 @@ class CfgModels {
     class IVBag_500ml: IVBagBase {};
     class IVBag_1000ml: IVBagBase {};
 
+    class ACE_grave: IVBagBase {};
```

Comment:
I'm not familiar with models.

@PabstMirror does this look okay?

Reply:
I think so,
I know it needs `sections[] = {"camo"};`
but it's getting that through inheritance

Reply:
Afaik, it's completely safe to do that.